packages/SwingSet/src/kernel/gc-actions.js

```diff
@@ -15,31 +15,39 @@ function parseAction(s) {
 export function processNextGCAction(kernelKeeper) {
   const allActionsSet = kernelKeeper.getGCActions();
 
-  function getRefCount(kref) {
-    // When we check for a re-exported kref, if it's entirely missing, that
-    // qualifies (to us) as a zero refcount.
-    const owner = kernelKeeper.ownerOfKernelObject(kref);
-    if (owner) {
-      return kernelKeeper.getObjectRefCount(kref);
+  function filterAction(vatKeeper, action, type, kref) {
```

Comment:
This probably could use a comment explaining why the actions are being filtered, since the verb "filter" is pretty generic, i.e., what the respective meanings of the pre-filtration and post-filtration collections are supposed to be. I think what's happening is it's throwing away the actions that it's not actually going to perform this time through, but I'm not 100% sure my understanding is correct.